docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Reactive_Messsaging_SmallRye.adoc

```diff
@@ -0,0 +1,130 @@
+:smallrye-reactie-messaging-version:    2.5
+[[MicroProfile_Reactive_Messaging_SmallRye]]
+= MicroProfile Reactive Messaging Subsystem Configuration
+----
+:smallrye-reactie-messaging-version:    2.5
+----
+
+
+Support for https://microprofile.io/project/eclipse/microprofile-reactive-messaging[Eclipse MicroProfile Reactive Messaging] is
+provided as a Tech Preview feature by the _microprofile-reactive-messaging-smallrye_ subsystem.
+
+[[required-extension-microprofile-reactive-messaging-smallrye]]
+== Required Extension
+
+This extension is *not* included in the standard configurations included in the WildFly distribution.
+
+You can add the extension to a configuration either by adding
+an `<extension module="org.wildfly.extension.microprofile.reactive-messaging-smallrye"/>`
+element to the xml or by using the following CLI operation:
+
+[source,options="nowrap"]
+----
+[standalone@localhost:9990 /] /extension=org.wildfly.extension.microprofile.reactive-messaging-smallrye:add
+{"outcome" => "success"}
+
+[standalone@localhost:9990 /] /subsystem=microprofile-reactive-messaging-smallrye:add
+{
+    "outcome" => "success",
+    "response-headers" => {
+        "operation-requires-reload" => true,
+        "process-state" => "reload-required"
+    }
+}
+----
+
+If provisioning your own server and including the `microprofile-reactive-messaging` layer, you will get the required modules and the extension and subsystem will be added to your configuration. If the modules coming from the `microprofile-reactive-messaging-kafka` Galleon layer are also present while the subsystem is enabled you will get the Kafka Connector functionality.
+
+To use this subsystem, you must also enable the <<MicroProfile_Reactive_Streams_Operators_SmallRye, MicroProfile Reactive Streams Operators>> extension and subsystem.
```

Comment:
Perhaps this sentence should go before the previous paragraph about layers.  AIUI if you use the layers the layer deps handle this for you. Discussing this after discussing layers implies you have to do something even with layers.